Makefile

```diff
@@ -165,10 +165,19 @@ taskmetadata-validator:
 ecr-execution-role-image:
 	$(MAKE) -C misc/ecr $(MFLAGS)
 
+.PHONY: gocyclo static-check
+gocyclo:
+	# Run gocyclo over all .go files in the agent, excluding vendor/ and testutils/ directories, and all *_test.go and *_mocks.go files
+	gocyclo -over 15 $(shell go list -f '{{$$p := .}}{{range $$f := .GoFiles}}{{$$p.Dir}}/{{$$f}} {{end}}' ./agent/... | grep -v /vendor/ | grep -v /testutils/ | grep -v _test\.go$ | grep -v _mocks\.go$)
```

Comment:
Can you break this up into multiple lines so it doesn't wrap so much?  You should be able to do that with `\`.

Reply:
broken up the line at the first pipe.